File: app/src/main/java/com/example/gadsleaderboard/adapters/SkillIQFragment.java
package com.example.gadsleaderboard.adapters;;

import android.os.Bundle
import android.view.View
import com.example.gadsleaderboard.UI.BoardActivityFragment


class SkillIQFragment : BoardActivityFragment() {

        override fun initAdapter(): BaseRecyclerViewAdapter<*, *> = LearningLeadersAdapter()

        override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
            super.onViewCreated(view, savedInstanceState)

            loadData()
        }

        private fun loadData() {

        }

}

File: app/src/main/java/com/example/gadsleaderboard/adapters/SkillIQFragment.kt
package com.example.gadsleaderboard.adapters

import android.os.Bundle
import android.view.View
import com.example.gadsleaderboard.UI.BoardActivityFragment
import com.example.gadsleaderboard.models.LearnerHours


class SkillIQFragment : BoardActivityFragment<LearnerHours>() {

    override fun initAdapter() = LearningLeadersAdapter()
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        loadData()
    }

    private fun loadData() {

    }

}

